content/_releases/22/34.md

```diff
@@ -0,0 +1,171 @@
+the Dashboard Runlog record only for 5 days in the Database.
+
+#### Breaking Changes
+
+Setting the `TASK_STAT_START_INDEX_TTL` environment variable to control the retention
+period of the Dashboard Runlog is recommended for the new installations of the platform.
+
+If your platform installation already has the retention period set in the Database
+prior to your installation of the platform {{ page.version }} version then your
+environment variable [will not be applied](https://www.mongodb.com/docs/v4.2/reference/command/createIndexes/#index-options). If you need to change the value you must
+change the Database index setup prior to the deployment of the current release.
+
+If you need to change the retention period of Dashboard Runlog in the future, remove
+the `expireAfterSeconds` option of the Database index holding the Dashboard Runlog records,
+set the value `TASK_STAT_START_INDEX_TTL` of environment variable and then deploy
+the platform version again.
+
+### Kubernetes 1.22
+
+Our platform is now compatible with the Kubernetes version `1.22`.
+
+### MongoDB TLS verification
+
+You can now enforce MongoDB TLS verification in all platform services by using
+newly introduced HELM3 chart configurations:
+
+*   `.Values.global.secrets.mongodbTlsCertificateKey` - secret name with the TLS certificate and key. If specified, will be mounted to the services and specified in the [`tlsCertificateKeyFile` connection option](https://mongodb.github.io/node-mongodb-native/3.6/api/global.html#MongoClientOptions).
+*   `.Values.global.secrets.mongodbTlsCA` - secret name with CA certificate to validate MongoDB server certificate on the client side. If specified, will be mounted to the services and specified in the [`tlsCAFile` connection option](https://mongodb.github.io/node-mongodb-native/3.6/api/global.html#MongoClientOptions).
+
+## OEM Related
+
+Information in this section is intended for our customers who use the
+[OEM version of the elastic.io platform](https://www.elastic.io/saas-embedded-integration/).
+
+### Improvements to white-lebel credential management
+
+Here we present further improvements to the recently introduced
+[White-label credential management](/releases/22/20/#white-labeled-credential-management) feature.
+Now you can use HTML inline frame or **iFrame** to embed the credentials.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Embedded Credentials Demo</title>
+  <style>html, body, iframe { height: 85%; width: 90%; margin: 0;}</style>
+</head>
+<body>
+  <iframe src="https://[eio_platform_domain]/embedded-credentials/[repoId]?workspaceId=[workspaceId]&ssoProviderType=[ssoProviderType]&ssoProviderId=[ssoProviderId]" />
+</body>
+</html>
+```
+
+The embed URL construction scheme is the same. Refer to the
+[22.20 release](/releases/22/20#how-to-generate-unique-url) for details.
+
+> Please note: if you opt to provide white-label credentials using iFrame advise
```

Comment:
NOTE: End Users must enable pop-ups in their browser settings in order to use the embedded credential management